### PR TITLE
P#50179 Update show or hide reference estate list

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

**Review by jayay — COMMENTED** [review]
Looks good in general, I commented with one question.

File: js/onoffice-reference-estate-select.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+jQuery(document).ready(function() {
```
[the remark this answers is not in the document]
Comment (hungnc89) [reply js/onoffice-reference-estate-select.js:1]: Hi @jayay. `checkbox.js` only handles between 2 checkboxes. I need to handle between select and checkbox

[the remark this answers is not in the document]
Comment (jayay) [reply js/onoffice-reference-estate-select.js:1]: Oh sorry, didn't think about that. Would you mind reformatting the Javascript file? (only use jQuery if necessary, use `const` and `let` strictly over `var`)

[the remark this answers is not in the document]
Comment (hungnc89) [reply js/onoffice-reference-estate-select.js:1]: @jayay .I changed the `var` to `const`. I think this should be handled by Jquery

**Comment by hungnc89** [comment]
@jayay I updated to Vanilla Js

**Review by jayay — APPROVED** [review]

**Comment by jayay** [comment]
Thanks. Could you please also resolve the conflicts?

**Comment by hungnc89** [comment]
@jayay I fixed it

**Comment by michaelarnold-dev** [comment]
@tung-le-esg Robert told me that you closed #2704963 that was a bug in P#50179. Did you prepare a fix for that? Robert only wants you to close the task in enterprise in case it has been fixed

**Comment by vu-vuong-esg** [comment]
> @tung-le-esg Robert told me that you closed #2704963 that was a bug in P#50179. Did you prepare a fix for that? Robert only wants you to close the task in enterprise in case it has been fixed

Not yet. I just started to implement.

**Pull request closed without merging**